src/Maker/MakeSerializerNormalizer.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Serializer\Serializer;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+final class MakeSerializerNormalizer extends AbstractMaker
+{
+    public static function getCommandName(): string
+    {
+        return 'make:serializer:normalizer';
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConf)
+    {
+        $command
+            ->setDescription('Creates a new serializer normalizer class')
+            ->addArgument('name', InputArgument::OPTIONAL, 'Choose a class name for your normalizer (e.g. <fg=yellow>UserNormalizer</>)')
+            ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeSerializerNormalizer.txt'))
+        ;
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator)
+    {
+        $normalizerClassNameDetails = $generator->createClassNameDetails(
+            $input->getArgument('name'),
+            'Serializer\\Normalizer\\',
+            'Normalizer'
+        );
```

Comment:
For the serializer encoder maker, we just put it into the `Serializer` directory (not a sub-directory). I'm interested in how many encoders / normalizers applications have, and whether we should put them in sub-directories, or just in Serializer (to keep things a bit smaller).

Reply:
I know and I don't like it.
I have the same opinion about form type.

It really prefer to use a sub namespace to keep things clean. 

I usually have between 0 to 10 normalizers and sometimes 1 encoder. 

Btw voters are put in a sub namespace too. 

It's just an extra directory...